tests/e2e/README.md

```diff
@@ -226,20 +230,102 @@ The full command to build the site will look as follows:
 TRAVIS_MARIADB_VERSION=10.5.3 TRAVIS_PHP_VERSION=7.4.5 WP_VERSION=5.4.1 npm run docker:up
 ```
 
-## Writing tests
+## Guide for writing e2e tests
+
+### Tools for writing tests
 
 We use the following tools to write e2e tests:
 
 - [Puppeteer](https://github.com/GoogleChrome/puppeteer) – a Node library which provides a high-level API to control Chrome or Chromium over the DevTools Protocol
 - [jest-puppeteer](https://github.com/smooth-code/jest-puppeteer) – provides all required configuration to run tests using Puppeteer
 - [expect-puppeteer](https://github.com/smooth-code/jest-puppeteer/tree/master/packages/expect-puppeteer) – assertion library for Puppeteer
 
-Tests are kept in `tests/e2e/specs` folder.
+Tests are kept in `tests/e2e/core-tests/specs/` folder.
```

Comment:
Adding WC core tests should follow the instructions in #27916. In other repos using our e2e packages the tests should be in `tests/e2e/specs/`.

Reply:
Thanks for this @rrennick! Corrected. Ready for another review.